mkdocs.yml

```diff
@@ -22,6 +22,7 @@ nav:
     - 'Target TPM hardware interfaces': hardware_interfaces.md
     - 'Current issues': issues.md
     - 'TPM modules pinout': tpm_modules_pinout.md
```

Comment:
Group tpm_modules_pinout.md and tpm_renders.md to - 'lpnTPM mechanical design'

Reply:
ok